src/EditorFeatures/VisualBasicTest/AutomaticCompletion/AutomaticLineEnderTests.vb

```diff
@@ -218,13 +218,15 @@ End Module
 </code>)
         End Sub
 
+        ' The test verifies the integrated behavior which keeps the space '_'.
+        ' This corresponds to the actual VS behavior.
         <WpfFact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)>
         Public Sub TestWithLineContinuation()
             Test(
 <code>
 Module M
     Sub Main()
-        Dim _
+        Dim _ 
```

Comment:
>Dim _  [](start = 8, length = 6)

This deals with the temporary fix: [https://github.com/dotnet/roslyn/pull/27027/files#diff-b30fde62bc06fe8b48c40f2a374a7ef7R227](https://github.com/dotnet/roslyn/pull/27027/files#diff-b30fde62bc06fe8b48c40f2a374a7ef7R227)